src/olympia/addons/tests/test_cron.py

```diff
@@ -55,11 +54,11 @@ def test_catchall(self):
     def test_appsupport(self):
         ids = Addon.objects.values_list('id', flat=True)
         cron._update_appsupport(ids)
-        assert AppSupport.objects.filter(app=amo.FIREFOX.id).count() == 4
+        assert AppSupport.objects.filter(app=amo.FIREFOX.id).count() == 3  # ??
 
         # Run it again to test deletes.
         cron._update_appsupport(ids)
-        assert AppSupport.objects.filter(app=amo.FIREFOX.id).count() == 4
+        assert AppSupport.objects.filter(app=amo.FIREFOX.id).count() == 3  # ??
```

Comment:
No idea why there's `3` now (or `4` before).

Reply:
`base/seamonkey` fixture had an addon compatible with Firefox. So you're all good.